hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -271,16 +271,30 @@ def _compute_line_ids(self):
                             [t.task_id for t in timesheet]):
                         tasks += [self.env['project.task']]
                     for task in tasks:
-                        lines |= self.env['hr_timesheet.sheet.line'].create(
-                            sheet._get_default_sheet_line(
-                                date=date,
-                                project=project,
-                                task=task,
-                                timesheets=timesheet.filtered(
-                                    lambda t: date == t.date
-                                    and t.task_id.id == task.id),
-                            ))
-            sheet.line_ids = lines
+                        data_key = (date, project, task)
+                        timesheet = timesheet.filtered(
```

Comment:
`timesheet = timesheet_pr.filtered(`

where `timesheet_pr` should be the one created in https://github.com/OCA/hr-timesheet/pull/207/commits/b5d1a7a7b08e73246a144b37a269fb5c1042bd40#diff-a48ce9b933da452cb1910c0e256932c7R267.

Reply:
this window can be minimized